data/packages/net.slothsoft.unity-extensions.yml

```diff
@@ -1,20 +1,20 @@
 name: net.slothsoft.unity-extensions
 displayName: Slothsoft's Unity Extensions
-description: Some C# Extension functions for common Unity problems.
+description: 'C# extension functions, attributes and editor tools for common Unity problems.'
 repoUrl: 'https://github.com/Faulo/UnityExtensions'
 parentRepoUrl: null
-licenseSpdxId: WTFPL
-licenseName: Do What The F*ck You Want To Public License
+licenseSpdxId: MIT
+licenseName: MIT License
 topics:
   - editor-enhancement
   - utilities
-hunter: ''
+hunter: Faulo
 gitTagPrefix: ''
 gitTagIgnore: ''
-minVersion: '2019.3'
-image: null
-readme: 'master:README.md'
+minVersion: '2020.3'
```

Comment:
The minVersion is the minimal version to build, not the minimal Unity version (and yes the previous value is wrong as well)

Reply:
Please set the minVersion to ''.